2021_12_12_084922_create_benefits_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateBenefitsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('benefits', function (Blueprint $table) {
+            $table->id();
+            $table->enum('name', ['WiFi', 'Garage', 'Autonomous heat sources']);
```

Comment:
Doesn't have sense to have enum, you are making this list not extensible

Reply:
change enum into string, and unique